modules/configure-proxy-on-an-existing-deployment.adoc

```diff
@@ -7,6 +7,13 @@
 
 To configure a proxy in an existing deployment, you must export the `proxy-config` secret as a YAML file, update your proxy configuration in that file, and upload it as a secret.
 
+[NOTE]
+====
+If you configured a global proxy on your {ocp} cluster,  the Operator Lifecycle Manager (OLM) automatically configures Operators that it manages with the cluster-wide proxy. However, you can also configure installed Operators to override the global proxy or inject a custom CA certificate.
```

Comment:
this could either be `If you have configured...` or `If you configure...` whichever makes more sense.